modules/telestion-application/src/main/java/de/wuespace/telestion/application/launcher/TelestionLauncher.java

```diff
@@ -0,0 +1,283 @@
+package de.wuespace.telestion.application.launcher;
+
+import de.wuespace.telestion.application.deployment.Deployment;
+import de.wuespace.telestion.application.loader.Loader;
+import io.vertx.core.CompositeFuture;
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.VertxOptions;
+import io.vertx.core.json.JsonObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Reference implementation of a {@link ConfigLauncher} and {@link DeploymentLauncher}.
+ * <p>
+ * It loads the configuration from {@code $PWD/telestion.json} by default, parses it with the
+ * {@link TelestionConfigParser} and runs all {@link Loader Loaders} configured in the {@code telestion.json}.
+ * <p>
+ * Use this launcher in your {@code build.gradle} as main class:
+ * <pre>
+ * {@code
+ * // [...]
+ *
+ * application {
+ *     mainClass.set('de.wuespace.telestion.application.launcher.TelestionLauncher')
+ * }
+ *
+ * // [...]
+ * }
+ * </pre>
+ *
+ * @author Ludwig Richter (@fussel178)
+ * @see ConfigLauncher
+ * @see DeploymentLauncher
+ * @see Launcher
+ * @see TelestionConfigParser
+ * @see Loader
+ */
+public class TelestionLauncher implements ConfigDeploymentLauncher<JsonObject> {
+
+	/**
+	 * Default path to the main configuration file of the {@link TelestionLauncher}.
```

Comment:
add link to javadoc of Path pls :)

(applies to multiple locations)

Reply:
Done in [`94f4808` (#504)](https://github.com/wuespace/telestion-core/pull/504/commits/94f48089b7c8f198c1c3b37f7b41d800e9686f58)